demo/src/components/ElementsAPI.tsx

```diff
@@ -1,4 +1,4 @@
-import '@stoplight/elements-core/styles/elements.scss';
+import '@stoplight/elements-core/styles.css';
```

Comment:
Whoa, whoa.

So does it mean that css endpoint changed for the users?

Aaah... I see that that's core.

Why is demo using core? Can we change that?